contents/blog/recommended-reading.md

```diff
@@ -0,0 +1,148 @@
+---
+date: 2023-08-16
+title: "Recommended reading from the PostHog team"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - joe-martin
+featuredImage: ../images/blog/super-hog-pink.png
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Guides
+---
+
+There are thousands of books which will tell you how to build a product and launch a new startup, but it can be hard to know which are actually worth reading and which are full of generic advice (or written by AI). If you’re looking for inspiration, where should you turn?
+
+The PostHog team includes a number of voracious readers — we even have [our own book club](/handbook/people/training)! — so here’s a collection of the books our teams recommend. 
+
```

Comment:
I'd honestly consider cutting in the intro. Headline should be all the intro this article needs.